Alicerce.xcodeproj/project.pbxproj

```diff
@@ -1541,19 +1541,19 @@
 /* End PBXResourcesBuildPhase section */
 
 /* Begin PBXShellScriptBuildPhase section */
-		0AB0562720B81D7600DC363F /* Check for /usr/include/CommonCrypto/CommonCrypto.h */ = {
+		0AB0562720B81D7600DC363F /* CommonCrypto */ = {
```

Comment:
Why are we keeping this? Xcode 9- support?

Reply:
To support Xcode 9, yes.

Reply:
nhec 😂